### PR TITLE
Symfony 3.2 compatibility

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

Follows #863

### CODE REVIEW

File: features/circular_reference.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@main
```
Comment (soyuka) [inline features/circular_reference.feature]: quick question, what does those stand for?

File: .travis.yml [hunk 2]
```diff
@@ -30,8 +29,8 @@ install:
   - if [[ $deps = 'low' ]]; then composer update --prefer-dist --no-progress --no-suggest --prefer-stable --prefer-lowest --ansi; fi
 
 script:
-  - if [[ $coverage = 1 ]]; then phpdbg -qrr -dmemory_limit=-1 "$(phpenv which phpunit)" --coverage-php build/cov/coverage-phpunit.cov; else phpunit; fi
-  - if [[ $coverage = 1 ]]; then phpdbg -qrr vendor/bin/behat --profile coverage; else vendor/bin/behat; fi
+  - if [[ $coverage = 1 ]]; then phpdbg -qrr -dmemory_limit=-1 vendor/bin/phpunit --coverage-php build/cov/coverage-phpunit.cov; else vendor/bin/phpunit; fi
+  - if [[ $coverage = 1 ]]; then tags=(main doctrine hal hydra integration jsonld security swagger); for tag in ${tags[@]}; do CURRENT_TAG=$tag phpdbg -qrr -dmemory_limit=-1 vendor/bin/behat --profile coverage --tags $tag; done; else vendor/bin/behat; fi
```
Comment (sroze) [inline .travis.yml]: + `~[all-the-other-tags]` ? ;)

Comment (dunglas) [inline .travis.yml]: I've pushed something smarter than the previous implem, thought.

**Comment by soyuka** [comment]
Wow! Nice job there!

**Pull request merged**